tensorflow/core/kernels/segment_reduction_ops.h

```diff
@@ -78,6 +78,24 @@ struct UnsortedSegmentSumFunctor: public UnsortedSegmentBaseFunctor<Device, T, I
                   typename TTypes<T, 2>::Tensor output);
 };
 
+// Functor for UnsortedSegmentSumWithDropNegativesOp.
+// 'output_rows': the number of output segments (unique segment ids in
```

Comment:
Remove quotes unless they're there for consistency.

Reply:
Thanks, updated.